client/src/hooks/helpers/useAvailableFundsGlm.ts

```diff
@@ -12,7 +12,7 @@ export default function useAvailableFundsGlm(): UseBalanceReturnType<{
   value: bigint;
 }> {
   const { address } = useAccount();
-  const { data: blockNumber } = useBlockNumber({ chainId: networkConfig.id, watch: true });
```

Comment:
This was resulting in fetching on each block, even when user left the tab open in the background. Biggest chunk of operations in our Alchemy is `eth_blockNumber`.